src/db/pyproject.toml

```diff
@@ -0,0 +1,13 @@
+[project]
+name = "ugbio_db"
+version = "1.1.0.dev0"
+requires-python = ">= 3.11"
+dependencies = [
+    "ugbio_core",
+    "pymongo>=4.8.0",
+]
+description = "Ultima Genomics Internal DB Access utils"
+authors = [
+    { name = "Ilya Soifer", email = "ilya.soifer@ultimagen.com" },
+]
+readme = "README.db.md"
```

Comment:
You can add scripts to the pyprojct.toml which can be used as entry points in the docker. This is very nice and clean, consider adding it here as well

Reply:
True. But I don't have scripts in this module.